core/tauri/src/scope/fs.rs

```diff
@@ -36,6 +36,14 @@ pub struct Scope {
   forbidden_patterns: Arc<Mutex<HashSet<Pattern>>>,
   event_listeners: Arc<Mutex<HashMap<Uuid, EventListener>>>,
   match_options: glob::MatchOptions,
+  original_scope: OriginalScope,
```

Comment:
lets not create a new type, just add two fields instead, `default_allowed` and `default_forbidden`